src/VirtualClient/VirtualClient.Contracts/Parser/DataTableExtensions.cs

```diff
@@ -161,37 +161,48 @@ public static DataTable ConvertToDataTable(string text, IList<KeyValuePair<int,
             {
                 try
                 {
-                    if (unitIndex >= 0)
+                    object value = row[valueIndex];
+                    if (value == null || value == DBNull.Value)
```

Comment:
Good catch, I didn't get an example of a null row. Would you think it's reasonable we report double.NaN?

Reply:
This will tell us we get a null cell, vs ignored, we will not know if we got a row or not

Reply:
I'd rather just ignore. It means the data was not good and we have the raw results to consult if we need.

Reply:
The SAP workloads has tons of dashes that we are reporting double.NaN. They indicated they much prefer double.NaN vs ignore. Because they can see that machine was there doing nothing.